lib/logging/logging_preflight.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -Euo pipefail
+
+LOGFILE=$PWD/tmp/progress.txt
+
+# This script runs before the container is composed.
+
+mkdir -p tmp/logging
```

Comment:
Was getting permission denied without the following:
```suggestion
mkdir -p tmp/logging
chmod a+w tmp/logging
```